code/script.js

```diff
@@ -31,17 +35,113 @@ const showMessage = (message, sender) => {
 }
 
 // Starts here
+
+//Bot  Default Greeting
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+showMessage(`Hello there, What's your name?`, 'bot')
+}
+
+// This will display the first answer (user name)
+const handleNameInput = (event) => {
+  event.preventDefault()
+  const name = nameInput.value
+  showMessage(`My name is ${nameInput.value}`, 'user')
+  setTimeout(() => showFoodOptions(name), 1000)
+}
+
+//Bot Question 2
+const showFoodOptions = () => {
+  showMessage(`Nice to meet you ${nameInput.value}! What fika would you like?`, 'bot')
+inputWrapper.innerHTML = `
+<button id="cinnamonrollBtn">Cinnamon roll</button>
+<button id="semlaBtn">Semla</button>
+<button id="prinsesscakeBtn">Prinsess cake</button>
+`
+//Add event listener for "cinnamon roll"
+document.getElementById("cinnamonrollBtn").addEventListener("click", () => {
+  showMessage('Cinnamon roll', 'user')
+  setTimeout(showQuantity,1000)
+})
+
+//Add event listener for "semla"
+document.getElementById("semlaBtn").addEventListener("click" , () => {
+  showMessage('Semla', 'user')
+  setTimeout(showQuantity,1000)
+})
+
+//Add event listener for "prinsess cake"
+document.getElementById("prinsesscakeBtn").addEventListener("click", () => {
+   showMessage('Prinsess cake' , 'user')
+   setTimeout(showQuantity,1000)
+})
+}
+
+//Bot Question 3
```

Comment:
Here you might could have commented more specific about the function and its purpose, in order for a reader to follow the code.